### PR TITLE
Document turn-key solution to deploy Kubernetes to IBM Cloud

### PR DESCRIPTION
**Pull request by patrocinio**

### CODE REVIEW

File: docs/getting-started-guides/ibm-softlayer.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
```
Comment (johndmulhausen) [inline docs/getting-started-guides/ibm-softlayer.md]: Thanks for making a new page! You'll also need to add it to the left navigation. Please find a spot in /_data/guides.yml

File: docs/getting-started-guides/ibm-softlayer.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+Follow this procedure to create a Kubernetes environment in IBM SoftLayer:
+
+* First clone the project [https://github.com/patrocinio/kubernetes-softlayer](https://github.com/patrocinio/kubernetes-softlayer)
+
+* Edit the kubernetes.cfg file to enter the following SoftLayer configuration
+
+    - USER
+    - API_KEY
+    - (Optional) DATACENTER: Check http://www.softlayer.com/data-centers and look at the Ping/Trace Route column for the code. For example, the code for speedtest.wdc01.softlayer.com is wdc01
```
Comment (johndmulhausen) [inline docs/getting-started-guides/ibm-softlayer.md]: - (Optional) DATACENTER: Check [http://www.softlayer.com/data-centers](http://www.softlayer.com/data-centers) and look at the Ping/Trace Route column for the code. For example, the code for speedtest.wdc01.softlayer.com is `wdc01`.

File: docs/getting-started-guides/ibm-softlayer.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    - (Optional) CPU: Define the number of CPIUs you want in each server
```
Comment (johndmulhausen) [inline docs/getting-started-guides/ibm-softlayer.md]: s/CPIUs/CPUs

**Comment by patrocinio** [comment]
Hello @johndmulhausen,

Thanks for reviewing my pull request.
I updated the navigation in _data/guides.yml

I appreciate if you could review my request again.

Thanks,

**Comment by kelseyhightower** [comment]
@patrocinio We've been thinking about linking to external content like this instead of hosting things here. Ideally we will move all the "getting started guides" out of the main repo and simply link to an external site under your complete control.

Would you be willing to host this content in a separate repo and just provide a link here?

**Comment by kelseyhightower** [comment]
@patrocinio After careful consideration I'm going to reject this change for the following  reasons:

- The doc for running Kubernetes on IBM cloud should live outside of this repo because we lack the bandwidth or expertise to keep it updated.

I'm going to suggest that this guide be hosted in a location under your control and allow us to link to it from here. This will give you full control over the content and allow kubernetes users to discover that it exists.

**Comment by patrocinio** [comment]
Hey @kelseyhightower Initially I created an external page at https://github.com/patrocinio/kubernetes-softlayer but was asked to incorporate the documentation to kubernetes.io. 

How can I provide the link to kubernetes.io?

**Comment by patrocinio** [comment]
This is the original PR:

https://github.com/kubernetes/kubernetes.github.io/pull/114

**Pull request closed without merging**